### PR TITLE
[REG-1667] Fix overlay resizing

### PR DESCRIPTION
**Pull request by rcornwall-reg**
Updated `OverlayCanvasV2` prefab and `RGBotEntry` prefab to correctly resize on all screen sizes

---

Find the pull request instructions [here](https://www.notion.so/regressiongg/Pull-Request-Process-0d57a7eb582a446983edfacafa406f1e?pvs=4)

Every reviewer and the owner of the PR should consider these points in their request (feel free to copy this checklist so you can fill it out yourself in the overall PR comment)

- [ ] The code is extensible and backward compatible
- [ ] New public interfaces are extensible and open to backward compatibility in the future
- [ ] If preparing to remove a field in the future (i.e. this PR removes an argument), the argument stays but is no longer functional, and attaches a deprecation warning. A linear task is also created to track this deletion task.
- [ ] Non-critical or potentially modifiable arguments are optional
- [ ] Breaking changes and the approach to handling them have been verified with the team (in the Linear task, design doc, or PR itself)
- [ ] The code is easy to read
- [ ] Unit tests are added for expected and edge cases
- [ ] Integration tests are added for expected and edge cases
- [ ] Functions and classes are documented
- [ ] Migrations for both up and down operations are completed
- [ ] A documentation PR is created and being reviewed for anything in this PR that requires knowledge to use
- [ ] Implications on other dependent code (i.e. sample games and sample bots) is considered, mentioned, and properly handled
- [ ] Style changes and other non-blocking changes are marked as non-blocking from reviewers

### CODE REVIEW

File: src/gg.regression.unity.bots/Runtime/Prefabs/RGOverlayCanvasV2.prefab [hunk 1]
```diff
@@ -792,7 +792,7 @@ GameObject:
   m_Icon: {fileID: 0}
   m_NavMeshLayer: 0
   m_StaticEditorFlags: 0
-  m_IsActive: 0
+  m_IsActive: 1
```
Comment (RG-nAmKcAz) [inline src/gg.regression.unity.bots/Runtime/Prefabs/RGOverlayCanvasV2.prefab:795]: This seems to now be active by default... I don't think that was intentional

**Review by RG-nAmKcAz — CHANGES_REQUESTED** [review]
Layout seems to work well, but overlay should be off by default

File: src/gg.regression.unity.bots/Runtime/Prefabs/RGOverlayCanvasV2.prefab (hunk 1 above)
Comment (rcornwall-reg) [reply src/gg.regression.unity.bots/Runtime/Prefabs/RGOverlayCanvasV2.prefab]: Correct - fixed in latest

**Review by RG-nAmKcAz — APPROVED** [review]

**Review by vontell — APPROVED** [review]
